string coordinate_system
WaypointArray trajectory
---
bool response